app/assets/stylesheets/views/_transition-landing-page.scss

```
@@ -36,19 +36,29 @@ $header-breakpoint: 960px;
     background-position: top;
     min-height: 500px;
   }
+
+  // HIDE BACKGROUND IMAGE ON IE
+
+  // IE 6, 7, 8
+  @media \0screen\,screen\9 {
```

This could be replaced with the condition comments class if you like:

```suggestion
  .lte-ie8 {
```

---

Do we have one for ie9?

---

Unfortunately we don't use one for IE9, but it is possible - I think it'd require [a change in static](https://github.com/alphagov/static/blob/master/app/views/layouts/govuk_template.html.erb#L3).